mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientServiceImpl.kt

```diff
@@ -114,14 +120,16 @@ public class GitHubClientServiceImpl(
     private fun searchIssuesOrPullRequests(
         username: Username,
         token: PersonalAccessToken,
+        lastSuccessfulUpdate: Timestamp,
         itemType: ItemType
     ): IssuesAndPullRequestsSearchResult =
         runBlocking {
             val response = client.get("https://api.github.com/search/issues") {
                 url {
                     parameters.append(
                         "q",
-                        "is:${itemType.value()} mentions:${username.value}"
+                        "is:${itemType.value()} mentions:${username.value} " +
```

Comment:
Please reduce the code nesting by extracting a request to GitHub into a separate type, with a convenient API.